tree/30_generic_methods/sharedfile_to_node.cf

```diff
@@ -127,10 +127,10 @@ bundle agent sharedfile_to_node(target_uuid, file_id, file_path, ttl)
       "${canonified_path}_signed" and => { fileexists("${file_path}.sign"), isnewerthan("${file_path}.sign","${file_path}") };
 
       # classes for _classes_combine_ifthenelse
-      "${old_class_prefix}_exist_true" expression => "${old_class_prefix}_exist_kept";
-      "${old_class_prefix}_exist_false" expression => "${old_class_prefix}_exist_repaired";
-      "${class_prefix}_exist_true" expression => "${class_prefix}_exist_kept";
-      "${class_prefix}_exist_false" expression => "${class_prefix}_exist_repaired";
+      "${old_class_prefix}_exist_true" expression => "${old_class_prefix}_exist_kept", scope => "namespace";
```

Comment:
Why is this needed? Comment says _classes_combine_ifthenelse defines the outcome classes.